source/algorithms.tex

```diff
@@ -88,6 +88,14 @@
   \tcode{Forward\-Iterator2},
   the template argument shall meet the
   \oldconcept{ForwardIterator} requirements\iref{forward.iterators}.
+\item
+  If an algorithm’s template parameter is named
```

Comment:
Unicode quote used?  rendered as "algorithmâĂŹs"

Reply:
Yes. The automatic check could be enhanced to detect this.